hedera-mirror-rest/__tests__/specs/contracts-05-specific-id-not-contract-id.spec.json

```
@@ -0,0 +1,21 @@
+{
+  "description": "Contract api calls for a specific contract using contract id, not found",
```

nit: This is more of an entity type mismatch
```suggestion
  "description": "Contract api calls for a specific contract using contract id, matching type not found",
```
Also maybe rename file to `...-not-contract-entity-type.spec.json` so it's more clear